docs/reference/commandline/stats.md

```diff
@@ -57,8 +57,46 @@ Running `docker stats` on multiple containers by name and id against a Windows d
     3f214c61ad1d        nanoserver          "cmd"               2 minutes ago       Up 2 minutes                            big_minsky
     9db7aa4d986d        windowsservercore   "cmd"               2 minutes ago       Up 2 minutes                            mad_wilson
     09d3bb5b1604        windowsservercore   "cmd"               2 minutes ago       Up 2 minutes                            affectionate_easley
-   
+
     PS E:\> docker stats 3f214c61ad1d mad_wilson
     CONTAINER           CPU %               PRIV WORKING SET    NET I/O             BLOCK I/O
     3f214c61ad1d        0.00%               46.25 MiB           76.3 kB / 7.92 kB   10.3 MB / 14.7 MB
-    mad_wilson          9.59%               40.09 MiB           27.6 kB / 8.81 kB   17 MB / 20.1 MB	
\ No newline at end of file
+    mad_wilson          9.59%               40.09 MiB           27.6 kB / 8.81 kB   17 MB / 20.1 MB
+
+## Formatting
+
+The formatting option (`--format`) will pretty print container output
+using a Go template.
+
+Valid placeholders for the Go template are listed below:
+
+Placeholder | Description
+---- | ----
+`.Container` | Container name or ID
+`.CPUPerc`   | CPU percentage
+`.MemUsage`  | Memory usage
+`.NetIO`     | Network IO
+`.BlockIO`   | Block IO
+`.MemPerc`   | Memory percentage (Available only on Linux)
+`.PIDs`      | PIDs number (Available only on Linux)
+
+
+When using the `--format` option, the `stats` command will either
+output the data exactly as the template declares or, when using the
+`table` directive, will include column headers as well.
+
+The following example uses a template without headers and outputs the
+`Container` and `CPUPerc` entries separated by a colon for all images:
+
+    $ docker stats --format "{{.Container}}: {{.CPUPerc}}"
```

Comment:
Can you add a `bash` language-hint, and add a space between the "command" and the "output"? We've seen that that makes it a bit clearer what the output is, and what the actually _command_ is. Also looks like the output of your example is missing the colon; So;

    ```bash
    $ docker stats --format "{{.Container}}: {{.CPUPerc}}"

    09d3bb5b1604:        6.61%
    9db7aa4d986d:        9.19%
    3f214c61ad1d:        0.00%
    ```